packages/nimble-components/src/icon-base/tests/icon-metadata.ts

```diff
@@ -292,6 +292,9 @@ export const iconMetadata: {
     IconHourglass: {
         tags: ['history', 'timer']
     },
+    IconIndent: {
```

Comment:
The naming convention is for the icon name to be a literal description of the visuals. Looking across the terms on our icons today I'd think triangle-right-bars and triangle-left-bars

I'd use "indent" and "outdent" as tags / higher-level concepts since they don't describe the visual itself.

Reply:
I had the same convention in mind but was swayed by the [icon naming section of the CONTRIBUTING doc](https://github.com/ni/nimble/blob/main/packages/nimble-tokens/CONTRIBUTING.md#icon-naming) which says "Prefer [FontAwesome](https://fontawesome.com/) names for icons." 

Of course, it also goes on to say "Minimize the use of metaphors; choose descriptive and unambiguous names instead" which could be contradictory. I think our convention needs to be more clear - if the Font Awesome name uses a metaphor, should we follow it or write our own literal description? 

I think @fredvisser has the clearest vision here so I'd be happy to delegate the decision to him. To start the conversation, here's a possible tweak to the convention:

1. If Font Awesome has an equivalent icon, use its name. Small tweaks for clarity are allowed (e.g. modifiers to distinguish similar icons)
2. If Font Awesome doesn't have an equivalent icon, use a description of the icon's appearance:
   1. if the appearance is unambiguously a physical object, use that object's name (key, gear)
   2. if the appearance is a metaphor that's unambiguously used for one thing and one thing only, use that thing's name (wifi, list)
   3. if the meaning might be ambiguous, describe the appearance literally (three dots, triangle with line)

Alternatively our policy could be "ask Fred what the name should be" 😆

Reply:
Maybe what's catching me more off guard is that our icons don't look as unambiguously like indent and outdent icons as the FontAwesome examples. I could see ours representing "add to list" or "remove from list" or things along that nature as well.

Reply:
> If Font Awesome has an equivalent icon, use its name.

From the PR that made that contributing change the icons that were added were all named very literally based on the visual. Maybe at that time it was assumed most icons in FontAwesome would be named that way. 

Maybe the guidance could be "Use the FontAwesome name if the name is a literal description of the visual. If the name is a metaphor of the visual it can be used if the visual we create is unambiguously, universally, and singularly represents that metaphor. Otherwise if the new visual could be used to represent multiple metaphors then create a literal name".

Reply:
Decided offline to match FontAwesome exactly (even when we don't love their name) to try to avoid having long debates about names. I'll update the CONTRIBUTING docs to say so.

I'll also reach out to Brandon to see if he can make the icon less ambiguous.

Reply:
Updated with the versions of the icons that got team signoff.